src/window.rs

```diff
@@ -943,12 +943,22 @@ impl Window {
         self.window.request_user_attention(request_type)
     }
 
-    /// Returns the current window theme.
+    /// Sets the current window theme.
```

Comment:
Could you also port a Wayland section from `set_csd_theme`? Since on Wayland the theme you have is from client side decorations.